.gitignore

```diff
@@ -31,7 +31,6 @@ npm-debug.log
 tmp
 .map
 Thumbs.db
-*.*-e
```

Comment:
Would you kindly enlighten? Why do we want to keep this _across commits_?

I thought we want to clean up after running release.sh. I.e. why do we want to check -e files into github in the first place?